### PR TITLE
Producer & Consumer API

### PR DESCRIPTION
**Pull request by nahguam**
# stream-registry PR

Producer & Consumer API

# PR Checklist Forms

- [ ] CHANGELOG.md updated
- [x] Reviewer assigned
- [x] PR assigned (presumably to submitter)
- [x] Labels added (enhancement, bug, documentation)

### CODE REVIEW

**Comment by neoword** [comment]
Can we add a comment to CHANGELOG?

File: graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/model/GraphQLInfrastructure.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.graphql.model;
+
+import java.util.List;
+
+import lombok.Builder;
```
Comment (neoword) [inline graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/model/GraphQLInfrastructure.java]: dreaming of kotlin.... l-)

Comment (nahguam) [reply graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/model/GraphQLInfrastructure.java]: Interesting you say that :) We were just talking about this today...

**Comment by nahguam** [comment]
FYI - please do not merge - WIP subject to change

**Comment by massdosage** [comment]
> FYI - please do not merge - WIP subject to change

You can use Draft Pull Requests (https://github.blog/2019-02-14-introducing-draft-pull-requests/) to make this intention clear.

**Comment by nahguam** [comment]
> > FYI - please do not merge - WIP subject to change
> 
> You can use Draft Pull Requests (https://github.blog/2019-02-14-introducing-draft-pull-requests/) to make this intention clear.

Nice.  Pity it seems you can't convert a PR back to a draft

**Comment by neoword** [comment]
Moving to 0.7.1. in preparation of 0.7.0 release

**Pull request merged**